db/migrate/20161012193544_add_education_benefits_submissions_table.rb

```diff
@@ -0,0 +1,14 @@
+class AddEducationBenefitsSubmissionsTable < ActiveRecord::Migration
+  def change
+    create_table :education_benefits_submissions do |t|
+      t.string(:region, null: false)
+      t.timestamps(null: false)
+
+      EducationBenefitsClaim::APPLICATION_TYPES.each do |application_type|
```

Comment:
I'd rather this migration explicitly have the types defined, so that if any are added or removed in the future, this migration doesn't have a different result than it does right now